src/AyonUsdResolver/CMakeLists.txt

```
@@ -1,111 +1,564 @@
+set(HOUDINI_SPDLOG_INCLUDE "$ENV{HFS}/toolkit/include")
+set(HOUDINI_SPDLOG_LIB_DIR "$ENV{HFS}/dsolib")
+
+# =========================================================
+# INCLUDE DIRECTORIES
+# =========================================================
+target_include_directories(${CORE_LIB_TARGET}
+    PUBLIC
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${USD_INCLUDE_DIR}
+        ${USD_BOOST_INCLUDE_DIR}
+        ${Python_INCLUDE_DIRS}
+
+        # AYON API
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/src/AyonCppApi
+
+        # Dev-tools
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/ayon-cpp-dev-tools/src
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/ayon-cpp-dev-tools/src/lib
+
+        # DCC-SPECIFIC INCLUDES (Houdini only)
+        # Maya includes are handled via target linking below
+        $<IF:$<BOOL:${_usd_is_houdini}>,${HOUDINI_SPDLOG_INCLUDE},>
+
+        # Other deps
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/cpp-httplib
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/json/include
+
+        ${Boost_INCLUDE_DIRS}
 )
-target_include_directories(${AR_AYONUSDRESOLVER_TARGET_LIB}
+
+target_include_directories(${PYTHON_MOD_TARGET}
     PUBLIC
-    ${AR_BOOST_INCLUDE_DIR}
-    ${AR_PYTHON_INCLUDE_DIR}
-    ${AR_PXR_INCLUDE_DIR}
+        ${CMAKE_CURRENT_SOURCE_DIR}
+        ${USD_INCLUDE_DIR}
+        ${USD_BOOST_INCLUDE_DIR}
+        ${Python_INCLUDE_DIRS}
+
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/src/AyonCppApi
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/ayon-cpp-dev-tools/src
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/ayon-cpp-dev-tools/src/lib
+
+        # DCC-SPECIFIC INCLUDES (Houdini only)
+        $<IF:$<BOOL:${_usd_is_houdini}>,${HOUDINI_SPDLOG_INCLUDE},>
+
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/cpp-httplib
+        ${CMAKE_SOURCE_DIR}/ext/ayon-cpp-api/ext/json/include
+
+        ${Boost_INCLUDE_DIRS}
 )
-set_target_properties(${AR_AYONUSDRESOLVER_TARGET_LIB} PROPERTIES PREFIX "")
 
-target_compile_definitions(${AR_AYONUSDRESOLVER_TARGET_LIB}
+# =========================================================
+# LIBRARY LINKING
+# =========================================================
+# 1. Base Linking (Common)
+target_link_libraries(${CORE_LIB_TARGET}
     PRIVATE
-        # USD Plugin Internal Namings
-        MFB_PACKAGE_NAME=${AR_AYONUSDRESOLVER_USD_PLUGIN_NAME}
+        Threads::Threads
+        AyonCppApi
+        AyonCppDevToolsLib
+        ${Python_LIBRARIES}
+        
+        # Core USD libs
+        USD::ar
+        USD::tf
+        USD::plug
+        USD::arch
 )
 
-target_link_libraries(${AR_AYONUSDRESOLVER_TARGET_LIB} AyonCppApi)
-target_link_libraries(${AR_AYONUSDRESOLVER_TARGET_LIB} AyonCppDevToolsLib)
+target_link_libraries(${PYTHON_MOD_TARGET}
+    PUBLIC
+        ${CORE_LIB_TARGET}
+        ${Python_LIBRARIES}
+        
+        # Core USD libs
+        USD::ar
+        USD::tf
+        USD::plug
+        USD::arch
+)cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
```

```
CMake Error at src/AyonUsdResolver/CMakeLists.txt:261:
  Parse error.  Expected a newline, got identifier with text
  "cmake_minimum_required".
```

I think this might be a typo. 
```suggestion
)
```

---

The whole code was there twice. Fixed now